src/Juvix/FrontendContextualise/ModuleOpen/Transform.hs

```diff
@@ -39,6 +39,7 @@ transformModuleOpenExpr (Old.OpenExpress modName expr) = do
       let NameSpace.List {publicL} = NameSpace.toList innerC
           newSymbs = fst <$> publicL
        in protectOpenPrim newSymbs $ do
+            -- TODO ∷ should we update this to not shad
```

Comment:
to not shadow?

Reply:
yes to now shadow the functions already in scope.